backend/app/models/dictionary_dto.py

```diff
@@ -8,3 +10,13 @@ class DictionaryDto(CamelModel):
     description: str
 
     model_config = ConfigDict(from_attributes=True)
+
+    valid_pattern: ClassVar[re.Pattern] = re.compile(r"^[\w\s\-]+$")
+
+    @field_validator("name", "description", mode="before")
```

Comment:
Avrei bisogno di un chiarimento su come funziona questo metodo. Inoltre, sarebbe da valutare se rinominare questo file. Perché teoricamente i DTO non dovrebbero contenere logica. Però non so come viene inteso un metodo di validazione.

Reply:
si è vero in generale però con questa libreria in realtà il funzionamento per automatizzarlo viene gestito dentro il DTO

Reply:
sì, infatti testando me ne sono accorto.

Reply:
Allora rimane solo da togliere il controllo sulla descrizione